src/scss/06_components/_tags.scss

```diff
@@ -0,0 +1,53 @@
+$tag-gap: $spacing-half;
```

Comment:
This could be a [private variable](https://sass-lang.com/documentation/at-rules/use#private-members) (like this: `$-tag-gap`)